src/components/Sidebar/Sidebar.tsx

```diff
@@ -109,7 +153,39 @@ const Sidebar: FC = () => {
                 value=""
               />
             )}
-
+            {nodeType === 'reference' && (
+              <div>
```

Comment:
To clean up this file the other node types have a an Editor component if its more than just a single line. I would suggest breaking this out into `ReferenceNodeEditor.tsx`